### PR TITLE
Fix: domain migration

### PR DESCRIPTION
**Pull request by zone117x**
Closes https://github.com/hirosystems/stacks-blockchain-api/issues/1595

* [x] toml references for seed nodes
* [x] API url references
* [x] Explorer url references

### CODE REVIEW

File: stacks-blockchain/Stacks-testnet.toml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 [node]
 rpc_bind = "0.0.0.0:20443"
 p2p_bind = "0.0.0.0:20444"
-bootstrap_node = "047435c194e9b01b3d7f7a2802d6684a3af68d05bbf4ec8f17021980d777691f1d51651f7f1d566532c804da506c117bbf79ad62eea81213ba58f8808b4d9504ad@testnet.stacks.co:20444"
+bootstrap_node = "029266faff4c8e0ca4f934f34996a96af481df94a89b0c9bd515f3536a95682ddc@seed.testnet.hiro.so:20444"
 # working_dir = "/tmp/stacks-blockchain-data"
```
Comment (zone117x) [inline stacks-blockchain/Stacks-testnet.toml:7]: @CharlieC3 what about the reference below to `bitcoind.testnet.stacks.co`, should these be updated to a `hiro.so` domain?

File: docs/openapi.yaml [hunk 2]
```diff
@@ -1,8 +1,8 @@
 openapi: 3.0.2
 servers:
-  - url: https://stacks-node-api.mainnet.stacks.co/
+  - url: https://api.mainnet.hiro.so/
     description: Mainnet
-  - url: https://stacks-node-api.testnet.stacks.co/
+  - url: https://api.testnet.hiro.so/
     description: Testnet
   - url: http://localhost:3999/
     description: Local
```
Comment (zone117x) [inline docs/openapi.yaml:8]: @LakshmiLavanyaKasturi @CharlieC3 what about the `docs.stacks.co` references in this file and in various markdown files under `content/`? Should these be replaced with `docs.hiro.so`?

**Review by CharlieC3 — APPROVED** [review]
Just calling out here that the Explorer URL changes are a bit ahead of the migration. The Explorer isn't planned to be migrated until April 10th. But it shouldn't be a big issue to make these updates now.

LGTM! :shipit:

**Comment by zone117x** [comment]
> @zone117x Could you also change the reference here? https://github.com/hirosystems/stacks-blockchain-api/blob/master/client/src/generated/runtime.ts#L16

This should be changed automatically during CI build and publish (since this is a generated file). Right now committing the generated client files results in a huge diff from lots of changed added since last time they were committed. For now I can do a one-off edit to that file. 

At some point we should either delete and not commit the generated client files, or do an isolated PR for the latest updates then enforce updates in new PRs somehow.

**Review by smcclellan — APPROVED** [review]

**Pull request merged**